pkg/helpers/azureconst.go

```diff
@@ -61,6 +61,8 @@ func GetAzureLocations() []string {
 		"usgovarizona",
 		"usgovtexas",
 		"francecentral",
+		"uaenorth",
```

Comment:
This file is auto generated and shouldn't be edited manually, you can add the new regions at  https://github.com/Azure/aks-engine/blob/ce3f9884c0d9c579f5a8ef78749b2d100d322298/pkg/helpers/generate_azure_constants.py#L107 instead and run `make generate`

Reply:
My bad, `generate_azure_constants.py` shouldn't be changed manually -- see @mboersma's comment above he got it right. 

In any case, the regions shouldn't be added manually into the file either, but running `make generate-azure-constants` should add them to azureconst.go and generate the same delta you have (if that's not already what you did).

Reply:
I don't think we need to modify the python script: it will pick up the two new regions as-is. That's a list of special regions that might not be visible depending on subscription. This PR already has the correct changes the python script would produce (except for some new VM types, but that doesn't need to be part of this addition).